src/orquestra/quantum/circuits/_builtin_gates.py

```diff
@@ -11,11 +11,13 @@
 GateRef = Union[_gates.Gate, GatePrototype]
 
 
-def make_parametric_gate_prototype(name, matrix_factory, num_qubits) -> GatePrototype:
+def make_parametric_gate_prototype(
+    name, matrix_factory, num_qubits, is_hermitian=False
```

Comment:
Shouldn't we add it to other places we use this constructor? Cause I guess there are other Hermitian gates as well?

Reply:
Other parametric gates that we had so far are rotations. The `is_hermitian` flag is used to indicate that the gate is hermitian no matter what its parameters are. Hence, it could be safely omitted so far.